website/docs/tutorials/react-patterns.md

```diff
@@ -84,6 +96,8 @@ function MyComponent() {
 }
 ```
 
+Note that we import `useLingui` component from `@lingui/macro`. There is also a runtime version of `useLingui` hook exported from `@lingui/react`. In the case above, it doesn't matter what version to choose since we use only `i18n` object which is presented in both.
```

Comment:
```suggestion
Note that we import `useLingui` from `@lingui/macro`. There is also a runtime version of `useLingui` hook exported from `@lingui/react`. In the case above, it doesn't matter what version to choose since we use only `i18n` object which is presented in both.
```